ocs_ci/helpers/dr_helpers.py

```diff
@@ -1083,3 +1084,117 @@ def verify_drpolicy_cli(switch_ctx=None):
         raise UnexpectedBehaviour(
             f"DRPolicy is not in succeeded or validated state: {status}"
         )
+
+
+def disable_dr_from_app(secondary_cluster_name):
+    old_ctx = config.cur_index
+    config.switch_acm_ctx()
+
+    # get all placement and replace value with surviving cluster
+    config.switch_acm_ctx()
+    placement_obj = ocp.OCP(kind=constants.PLACEMENT)
+    placements = placement_obj.get(all_namespaces=True).get("items")
+    for placement in placements:
+        name = placement["metadata"]["name"]
+        if (name != "all-openshift-clusters") and (name != "global"):
+            namespace = placement["metadata"]["namespace"]
+            path = "/spec/predicates/0/requiredClusterSelector/labelSelector/matchExpressions/0/values/0"
+            params = f"""[{{"op": "replace", "path": "{path}", "value": "{secondary_cluster_name}"}}]"""
+            # placement_obj.patch(resource_name=name, params=params, format_type="json")
+            cmd = f"oc patch placement {name} -n {namespace}  -p '{params}' --type=json"
+            run_cmd(cmd)
+
+    '''
+    placements = placement_obj.get(all_namespaces=True).get("items")
+    for placement in placements:
+        namespace = placement["metadata"]["namepsace"]
+        name = placement["metadata"]["name"]
+        placements_dict = dict()
+        if namespace != "all-openshift-clusters":
+            placements_dict.update({name: namespace})
+
+    for name, namespace in placements_dict.items():
+        patch_cmd = f"""[{ "op": "replace",
+                    "path": "/spec/predicates/0/requiredClusterSelector/labelSelector/matchExpressions/0/values/0",
+                    "value": {secondary_cluster_name}}]"""
+        cmd = f"oc patch placement {name} --type=json -p '{patch_cmd}' -n namespace"
+        run_cmd(cmd)
+    '''
+
+    # Delete all drpc
+    run_cmd("oc delete drpc --all -A")
+
+    # Remove annotation from placements
+    for placement in placements:
+        name = placement["metadata"]["name"]
+        if (name != "all-openshift-clusters") and (name != "global"):
+            namespace = placement["metadata"]["namespace"]
+            path = "/metadata/annotations/cluster.open-cluster-management.io~1experimental-scheduling-disable"
+            params = f"""[{{"op": "remove", "path": "{path}"}}]"""
+            placement_obj.patch(resource_name=name, params=params, format_type="json")
+
+    config.switch_ctx(old_ctx)
+    return placement_obj
+
+
+def replace_cluster(workload, primary_cluster_name, secondary_cluster_name):
+
+    """
+    Function to do core replace cluster task
+
+    Args:
+    workload(List): List of workload objects
+    primary_cluster_name (str): Name of the primary DRcluster
+    secondary_cluster_name(str): Name of the secondary DRcluster
+
+    """
+
+    # Delete dr cluster
+    config.switch_acm_ctx()
+    run_cmd(cmd=f"oc delete drcluster {primary_cluster_name} --wait=false")
```

Comment:
validate drcluster is removed.

Reply:
As per replace cluster doc drcluster will not remove here.
```Delete the DRCluster for the replacement cluster.

$ oc delete drcluster <drcluster_name> --wait=false
Note
Use --wait=false since the DRCluster will not be deleted until a later step.```